src/locales/scripts/get-translations.js

```diff
@@ -95,11 +98,16 @@ const fetchAndConvertJed1xTranslations = (locales) => {
   return Promise.allSettled(locales.map(fetchJed1xTranslation))
     .then((res) => {
       let successfulTranslations = []
+      let failedTranslations = []
       res.forEach(({ status, value }, index) => {
         if (status === "fulfilled" && !isEmpty(value)) {
           successfulTranslations[locales[index]] = value
+        } else {
+          failedTranslations.push(`${locales[index]} (${value})`)
         }
       })
+      if (failedTranslations.length)
+        console.log(`Failed to fetch ${failedTranslations.join(", ")}`)
```

Comment:
```suggestion
      if (failedTranslations.length) {
        console.log(`Failed to fetch ${failedTranslations.join(", ")}`)
      }
```

Since prettier is wrapping this to two lines, anyway, might as well bracket it.